binary package not working: cmake_modules_DIR not necessarily set

In a clean workspace (with no package depending on cmake_modules), the include ist still not found:

```
CMake Error at [...]/CMakeLists.txt:64 (include):
  include could not find load file:

    CodeCoverage
```

Adding a debug message shows, that `${cmake_modules_DIR}`.

Pull request follows...

--- a/cmake/code_coverage-extras.cmake.installspace.in
+++ b/cmake/code_coverage-extras.cmake.installspace.in
@@ -1,2 +1,2 @@
 # Append the installed cmake modules to the cmake module path
-list(APPEND CMAKE_MODULE_PATH "${cmake_modules_DIR}/../../../@CATKIN_PACKAGE_SHARE_DESTINATION@/cmake/Modules")
+list(APPEND CMAKE_MODULE_PATH "${code_coverage_DIR}/../../../@CATKIN_PACKAGE_SHARE_DESTINATION@/cmake/Modules")

